### PR TITLE
cut v1.2.0

### PR DESCRIPTION
**Pull request by nojnhuh**

### CODE REVIEW

**Review by keithmattix — APPROVED** [review]

**Review by zr-msft — APPROVED** [review]
👍

File: config.toml [hunk 1]
```diff
@@ -89,7 +89,7 @@ project_home = "https://openservicemesh.io/"
 
 # Version params
 osm_branch = "release-v1.2"
-osm_version = "v1.2.0-rc.1"
+osm_version = "v1.2.0"
```
Comment (shashankram) [inline config.toml:92]: The latest version needs to be updated In L96

Comment (nojnhuh) [reply config.toml:92]: @zr-msft has been making those changes in the rest of the release branches, but not sure I remember seeing one for the release-v1.2 branch. @zr-msft can you handle that if you don't already have a PR open?

Comment (shashankram) [reply config.toml:92]: This hasn't been done on `main` either: https://github.com/openservicemesh/osm-docs/blob/main/config.toml
Could one of you update this ASAP so `docs.openservicemesh.io` correctly redirects to the release-v1.2 landing page and the dropdowns are accurate?

Thanks

**Pull request merged**